data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_to_damage_type_special.cfg

```diff
@@ -0,0 +1,195 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [resistance]apply_to= when opponent use [damage_type]alternative_type=
+##
+# Actions:
+# Give all units vulneraility to arcane damage with a value of -100%
+# Attack each other with blade,arcane weapons
+##
+# Expected end state:
+# The damage from the attack is not changed because a bug in[damage_type] not resolved before stable branch
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "damage_secondary_type_with_resistance_ability_test" (
+    [event]
+        name=start
+        [modify_unit]
+            [filter]
+            [/filter]
+            max_hitpoints=100
+            hitpoints=100
```

Comment:
All of this `[modify_unit]` is unneeded, the ATTACK_AND_VALIDATE handles it.

Reply:
test changed now and it's work(fir Ubuntu)